src/views/Predictions/components/Label.tsx

```diff
@@ -89,7 +90,7 @@ const Label = styled(Flex)<{ dir: 'left' | 'right' }>`
 `
 
 export const PricePairLabel: React.FC = () => {
-  const price = useGetLastOraclePrice()
+  const price = usePollOraclePrice()
   const priceAsNumber = parseFloat(formatBigNumberToFixed(price, 3, 8))
   const { countUp, update } = useCountUp({
```

Comment:
this hook makes this component render too much when price changes, which calls usePollOraclePrice() hook too many times in one second. Although it probably will not make a onchain request in short time, retrieving data from cache make performance suffer?

Reply:
I prefer not to split fetch hook and get cache hook here.I will add `dedupingInterval` to prevent some edge cases.

Actually I thought we could use event listener to do that, but it seems like the event is not implement in contract side